flow-typed/npm/lodash_v4.x.x.js

```diff
@@ -3114,9 +3114,9 @@ declare module "lodash/fp" {
     defaultTo<T1: void | null, T2>(defaultValue: T2): (value: T1) => T2;
     defaultTo<T1: void | null, T2>(defaultValue: T2, value: T1): T2;
     flow: $ComposeReverse;
-    flow(funcs: Array<Function>): Function;
+    flow(...funcs: Array<Function>): Function;
```

Comment:
this should not be edited directly but using `flow-typed install lodash`. it should be up to date and the file header signature should change so.

tell me if you did it but signature not changing is the source of the issue imo

Reply:
adrien me dit que ca va tout casser ds le typage flow si on fait cette modif, du coup ca ne sera pas inclus dans cette PR si ca te va ;)

Reply:
pour préciser, le pipe qui accepte pas un spread, c'est clairement un vrai bug.
Qui en plus jette tout discredit sur la qualité de ce flow-typed. (ya des types qui n'ont jamais été utilisé en pratique, voir typer).

Et puis, j'avais au préalable tester de mettre à jour flowbin dans #1638  . ça casse tout votre typage flow. (89e860e)

Reply:
(Et je passe la question d'investir du temps sur flow, et notre typage flow)